RLTest/redis_std.py

```diff
@@ -180,7 +180,8 @@ def createCmdArgs(self, role):
         if self.password:
             cmdArgs += ['--requirepass', self.password]
         if self.clusterEnabled and role is not SLAVE:
-            cmdArgs += ['--cluster-enabled', 'yes', '--cluster-config-file', self._getFileName(role, '.cluster.conf'),
+            # creating .cluster.conf in /tmp as lock fails on NFS
```

Comment:
@rafie im not sure where this comes from, does this fix something specific?

Reply:
Yes, it fixes a crash I have when running on NFS. I was thinking about introducing a command-line option to trigger that, but since command-line options currently do not infiltrate into `redis_std.py`, I just fixed it this way.

Reply:
@rafie IIUC this means we assume this file is on /tmp. but previous it wasnt assuming that. what do I miss?

Reply:
This file is created by RLTest, it's not something we provide. No?